app/views/admin/settings/index.html.slim

```diff
@@ -0,0 +1,27 @@
+- content_for(:title, 'Settings')
```

Comment:
It makes sense to have a settings page and it seems like a lot of the stuff under the Division page (e.g. colors, currency, locales, notify on new logs) should move here. @rgalusca what do you think? Obviously this would be a separate issue...